appengine/pond/duck/style.css

```diff
@@ -24,6 +24,13 @@
   border: 1px solid #ddd;
 }
 
+#leaderboard {
+  position: fixed !important;
+  height: auto !important;
+  bottom: 10px;
+  border: 1px solid #ddd;
```

Comment:
We haven't been following this consistently as we should, but these should be alphabetized (based on style guide)